### PR TITLE
[17.x][WFLY-11786][WFLY-12175][WFLY-12277] Port doc fixes to 17.0.x branch

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.jboss.org/browse/WFLY-11786
https://issues.jboss.org/browse/WFLY-12175
https://issues.jboss.org/browse/WFLY-12277

Port recent doc fixes to the 17.0.x branch. The basic point being that then the docs can be generated, with '17' as the version, and then that output can be copied to the '17' dir of a checkout  wildfly/wildlfy.github.io in order to create a PR to fix the actual WF 17 docs.

### CODE REVIEW

File: docs/src/main/asciidoc/Getting_Started_Guide.adoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ WildFly team;
 :doctype: book
 :icons: font
 :source-highlighter: coderay
+ifdef::env-github[:imagesdir: images/]
```
Comment (jamezp) [inline docs/src/main/asciidoc/Getting_Started_Guide.adoc]: Not really an issue for this PR, but I wonder if we need any of this `ifdef::env-github` stuff as we generate the documentation.

**Review by jamezp — APPROVED** [review]

File: docs/src/main/asciidoc/Getting_Started_Guide.adoc (hunk 1 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/Getting_Started_Guide.adoc]: @jamezp I'll conduct a test using this PR branch...

Comment (bstansberry) [reply docs/src/main/asciidoc/Getting_Started_Guide.adoc]: @jamezp It does what I thought it does. It allows the images to be properly rendered when looking at the adoc pages in wildfly/wildfly github repo.

Compare section 1.4.4 of this:

https://github.com/bstansberry/wildfly/blob/doc_fixes/docs/src/main/asciidoc/Getting_Started_Guide.adoc

with the same from a branch I just created where I removed the ifdef from the page:

https://github.com/bstansberry/wildfly/blob/doc_fixes/docs/src/main/asciidoc/Getting_Started_Guide.adoc

The latter is missing the big graphic showing the WildFly welcome page.

Comment (jamezp) [reply docs/src/main/asciidoc/Getting_Started_Guide.adoc]: Ah that makes sense.

**Pull request merged**